Blahgger/Blahgger/Views/Accounts/Create.cshtml

```diff
@@ -0,0 +1,43 @@
+﻿@model Blahgger.Models.User
+
+@{
+    ViewBag.Title = "Create";
+}
+
+<h2>Create</h2>
+
+@using (Html.BeginForm()) 
+{
+    @Html.AntiForgeryToken()
```

Comment:
Don't forget to add the `ValidateAntiForgeryToken` attribute to your POST action method.